src/assets/i18n/en.json5

```diff
@@ -2474,7 +2474,7 @@
 
   "item.preview.dc.identifier": "Identifier:",
 
-  "item.preview.dc.relation.ispartof": "Journal or Serie",
+  "item.preview.dc.relation.ispartof": "Journal or Serial",
```

Comment:
@misilot : I think it's meant to say "Series"  ... i.e. this Item is part of a "Journal or Series".  So, I think it was just missing an "s" on the end.

Reply:
whoops, that makes more sense! let me fix that